zerver/openapi/zulip.yaml

```diff
@@ -790,6 +790,71 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/NonExistingStreamError'
+  /users/me/subscriptions/muted_topics:
+    patch:
+      description: Toggle muting for a specific topic the user is subscribed
+        to.
+      parameters:
+      - name: stream
+        in: query_function
+        description: The name of the stream where the topic is located.
+        schema:
+          type: string
+        example: Verona
+        required: true
+      - name: topic
+        in: query
+        description: The name of the topic to (un)mute. Please note that this
+          does not have to exist in the stream in order for the query to
+          succeed.
```

Comment:
"Please note that this will succeed regardless of whether any messages have been sent to the specified topic in the past."